### DISCUSSION TITLE
How to use the collision component

### DISCUSSION THREAD

**Discussion by jrojasr1986** [visose/Robots#283] [Q&A] [answered]
I am trying to to use the collision component but not sure how to use it. I include the 7 meshes that my robot has including the tool) and consider the environment plan index as 7 assuming that is the base. Also, try 0, and seems that the result does not change if I change this value. But returns that the first target is the collision and that 2 meshes collide but are just one and a different one. Could you please help me correct my logic? How should I actually use the component? 

![image](https://github.com/visose/Robots/assets/130080096/d66bb2be-c851-4b9a-ba97-905152847c1a)

![image](https://github.com/visose/Robots/assets/130080096/862c73ef-cee8-4a08-a42d-864c93fa0442)

**Comment by visose** [comment, marked as answer]
Hi, I just added a "collisions" section in the wiki, let me know if it clarifies how it works.

**Reply by jrojasr1986** [reply]
Hello, thank you so much for your help. Yes, now it is clear.
